pkg/authorization/authorizer/authorizer.go

```diff
@@ -3,30 +3,38 @@ package authorizer
 import (
 	"errors"
 
-	kerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
 	"k8s.io/apiserver/pkg/authorization/authorizer"
-
-	"github.com/openshift/origin/pkg/authorization/rulevalidation"
+	kauthorizer "k8s.io/apiserver/pkg/authorization/authorizer"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+	authorizerrbac "k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac"
 )
 
 type openshiftAuthorizer struct {
-	ruleResolver          rulevalidation.AuthorizationRuleResolver
+	delegate              kauthorizer.Authorizer
 	forbiddenMessageMaker ForbiddenMessageMaker
 }
 
-func NewAuthorizer(ruleResolver rulevalidation.AuthorizationRuleResolver, forbiddenMessageMaker ForbiddenMessageMaker) (authorizer.Authorizer, SubjectLocator) {
-	ret := &openshiftAuthorizer{ruleResolver, forbiddenMessageMaker}
-	return ret, ret
+type openshiftSubjectLocator struct {
```

Comment:
is this just an intermediate step?  I'm surprised we still have a wrapper. If it is intermediate, please open an issue to remove it.